scripts/exregional_run_prepstart_ensmean.sh

```diff
@@ -158,11 +158,17 @@ while [[ $imem -le ${NUM_ENS_MEMBERS} ]];
 #
 # get ensemble mean
 #
-ncea fv_core.res.tile1.nc_mem* fv_core.res.tile1.nc
-ncea fv_tracer.res.tile1.nc_mem* fv_tracer.res.tile1.nc
-ncea sfc_data.nc_mem* sfc_data.nc
-ncea fv_srf_wnd.res.tile1.nc_mem* fv_srf_wnd.res.tile1.nc
-ncea phy_data.nc_mem* phy_data.nc
+#ncea fv_core.res.tile1.nc_mem* fv_core.res.tile1.nc
+#ncea fv_tracer.res.tile1.nc_mem* fv_tracer.res.tile1.nc
+#ncea sfc_data.nc_mem* sfc_data.nc
+#ncea fv_srf_wnd.res.tile1.nc_mem* fv_srf_wnd.res.tile1.nc
+#ncea phy_data.nc_mem* phy_data.nc
+#
```

Comment:
Is it safe to remove line 161-166?

Reply:
This is a temporary solution to skip ensemble mean calculation. For now, simply use member 1 to get the reference state O-B. I will add the ensemble mean calculation later using OU code.

Reply:
Got it. But you will no longer use ncea, right?